azurelinuxagent/common/cgroup.py

```diff
@@ -241,11 +243,16 @@ def get_throttled_time(self):
         return float(self._current_throttled_time - self._previous_throttled_time) / 1E9
 
     def get_tracked_metrics(self, **kwargs):
-        tracked = [
-            MetricValue(MetricsCategory.CPU_CATEGORY, MetricsCounter.PROCESSOR_PERCENT_TIME, self.name, self.get_cpu_usage()),
-        ]
+        tracked = []
+        cpu_usage = self.get_cpu_usage()
```

Comment:
updating previous/current values happens inside these get functions on every poll, so no explicit logic needed to reset the values.